### PR TITLE
CMOS-266: Add Memcached Time Jump Alert

### PR DESCRIPTION
**Pull request by charlie-hayes**

### CODE REVIEW

File: docs/modules/ROOT/partials/cmos-health-checks.adoc [hunk 1]
```diff
@@ -122,6 +122,18 @@ However, repeated crashes should be investigated as they may be caused by an und
...
+=== Memcached Time Jumps (CB90057)
+
+*Background*: Time jumps occur when the cluster manager process (`beam.smp`) has not been scheduled by the CPU, or not scheduled enough for a significant period of time.
+Time jumps are the result of underlying issues (e.g. over provisioning or VM resource contention) with the machine that Couchbase Server is running on, particularly in virtualised environments.
+
+*Condition*: Memcached detected a time jump.
+
+*Remediation*: Check for evidence of your node being over provisioned or for evidence of VM resource contention.
+
+*Further Reading*:
```
Comment (redm2) [inline docs/modules/ROOT/partials/cmos-health-checks.adoc:134]: Did you forget to put a link in here?

**Review by redm2 — APPROVED** [review]
Looks fine to me

File: docs/modules/ROOT/partials/cmos-health-checks.adoc (hunk 1 above)
Comment (charlie-hayes) [reply docs/modules/ROOT/partials/cmos-health-checks.adoc:134]: 🤦

File: .aspell.en.pws [hunk 2]
```diff
@@ -81,4 +81,5 @@ XDCR
 Eventing
 Analytics
 vBucket
-vBuckets
\ No newline at end of file
+vBuckets
+virtualised
```
Comment (markspolakovs) [inline .aspell.en.pws:85]: Put a completely blank line at the end of this file, so we don't get bogus diffs when someone adds a new word

File: docs/modules/ROOT/partials/cmos-health-checks.adoc [hunk 3]
```diff
@@ -122,6 +122,17 @@ However, repeated crashes should be investigated as they may be caused by an und
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/services-and-indexes/services/analytics-service.html[Analytics Service]
 
+=== Memcached Time Jumps (CB90057)
+
+*Background*: Time jumps occur when the cluster manager process (`beam.smp`) has not been scheduled by the CPU, or not scheduled enough for a significant period of time.
```
Comment (markspolakovs) [inline docs/modules/ROOT/partials/cmos-health-checks.adoc:127]: `beam.smp` or `memcached`?

Comment (charlie-hayes) [reply docs/modules/ROOT/partials/cmos-health-checks.adoc:127]: This is just going off this line:
>These occur when the cluster manager process (beam.smp) has not been scheduled by the CPU, or not scheduled enough for a significant period of time.

In this support ticket: [40646](https://couchbasesupport.zendesk.com/agent/tickets/40646)

Comment (markspolakovs) [reply docs/modules/ROOT/partials/cmos-health-checks.adoc:127]: There's two different kinds of time jumps, `memcached` and `ns_server` - the LogQL you have here will find `memcached`'s, not `ns_server`'s.

**Review by markspolakovs — APPROVED** [review]

**Pull request merged**